soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -1134,4 +1134,78 @@ struct smb_fifo_reg {
 #define NPCX_SMBRXF_CTL_RX_THR           FIELD(0, 6)
 #define NPCX_SMBRXF_CTL_LAST             7
 
+/*
+ * Tachometer (TACH) Sensor device registers
+ */
+struct tach_reg {
+	/* 0x000: Timer/Counter 1 */
+	volatile uint16_t TCNT1;
+	/* 0x002: Reload/Capture A */
+	volatile uint16_t TCRA;
+	/* 0x004: Reload/Capture B */
+	volatile uint16_t TCRB;
+	/* 0x006: Timer/Counter 2 */
+	volatile uint16_t TCNT2;
+	/* 0x008: Clock Prescaler */
+	volatile uint8_t TPRSC;
+	volatile uint8_t reserved1;
+	/* 0x00A: Clock Unit Control */
+	volatile uint8_t TCKC;
+	volatile uint8_t reserved2;
+	/* 0x00C: Timer Mode Control */
+	volatile uint8_t TMCTRL;
+	volatile uint8_t reserved3;
+	/* 0x00E: Timer Event Control */
+	volatile uint8_t TECTRL;
+	volatile uint8_t reserved4;
+	/* 0x010: Timer Event Clear */
+	volatile uint8_t TECLR;
+	volatile uint8_t reserved5;
+	/* 0x012: Timer Interrupt Enable */
+	volatile uint8_t TIEN;
+	volatile uint8_t reserved6;
+	/* 0x014: Compare A */
+	volatile uint16_t TCPA;
+	/* 0x016: Compare B */
+	volatile uint16_t TCPB;
+	/* 0x018: Compare Configuration */
+	volatile uint8_t TCPCFG;
+	volatile uint8_t reserved7;
+	/* 0x01A: Timer Wake-Up Enablen */
+	volatile uint8_t TWUEN;
+	volatile uint8_t reserved8;
+	/* 0x01C: Timer Configuration */
+	volatile uint8_t TCFG;
+	volatile uint8_t reserved9;
+};
+
+/* TACH register fields */
+#define NPCX_TMCTRL_MDSEL_FIELD          FIELD(0, 3)
```

Comment:
Group this with the other TMCTRL bit defintions below.

Reply:
Done.